recipes/palantir-sdk/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file isn't needed as it's in the souce files (you still need the key in `meta.yaml`).